### PR TITLE
Gherkin Script:Getting Started - Spring Boot-ODC-4389

### PR DESCRIPTION
**Pull request by sanketpathak**
Problem: Need an easy getting started experience for Spring Boot developers.
Story: https://issues.redhat.com/browse/ODC-5442
Epic: https://issues.redhat.com/browse/ODC-4389

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature [hunk 1]
```diff
@@ -0,0 +1,157 @@
+Feature: UploadJAR file
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:1]: Can you please add @add-flow at the top of the file? WDYT @makambalaji

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+
+        @regression @manual
+        Scenario Outline: Upload JAR file from Add options
+            Given user is at the Add page
+             When user clicks on the Upload JAR file card
+              And user clicks on Browse in JAR file section
+              And user selects file to upload
+              And user selects appropriate Build image version
+              And user gives Application name as "sample-upload-app" and Name as "<name>"
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:17]:
```suggestion
              And user gives Application Groupings name as "sample-upload-app" and workload Name as "<name>"
```

Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:17]: It makes more sense and won't confuse anyone. WDYT?

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                  | sample-yaml-upload-1 | Deployment        |
+                  | sample-yaml-upload-2 | Deployment Config |
+
+        @regression @manual
+        Scenario Outline: Upload JAR file through drag and drop from Add options
+            Given user is at the Add page
+             When user clicks on the Upload JAR file card
+              And user drag and drop the file in JAR file section
+              And user selects appropriate Build image version
+              And user gives Application name as "sample-upload-app" and Name as "<name>"
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:35]: same as above

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+              And user selects resource type as "<resource_type>"
+              And user clicks on Create
+             Then user will be taken to Topology page
+              And user can see a toast notification of Jira file uploading with link to build logs
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:39]:
```suggestion
              And user can see a toast notification of Jar file uploading with link to build logs
```

File: frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature [hunk 5]
```diff
@@ -172,3 +172,86 @@ Feature: Topology chart area
...
+
+        @regression, @manual
+        Scenario: Drag and drop jar file in topology chart view
+            Given user has a jar file named "sample_yaml_upload.yaml"
+              And user is at the Topology page
+             When user drags and drop jar file on topology
+              And user gives Application name as "sample-upload-app" and Name as "sample-yaml-upload"
+              And user clicks on Create in Upload JAR file form
+             Then user is taken to topology
+              And user can see a toast notification of Jira file uploading with link to build logs
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature:197]:
```suggestion
              And user can see a toast notification of Jar file uploading with link to build logs
```

File: frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature [hunk 6]
```diff
@@ -172,3 +172,86 @@ Feature: Topology chart area
...
+             When user right clicks on the empty chart area
+              And user clicks on Add to Project
+              And user clicks on Upload JAR file
+              And user clicks on Browse in JAR file section
+              And user selects file to upload
+              And user clicks Clear and reupload the file
+              And user gives Application name as "sample-upload-app" and Name as "sample-yaml-upload-1"
+              And user clicks on Create
+             Then user is taken to topology
+              And user can see a toast notification of Jira file uploading with link to build logs
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature:213]:
```suggestion
              And user can see a toast notification of Jar file uploading with link to build logs
```

File: frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature [hunk 7]
```diff
@@ -172,3 +172,86 @@ Feature: Topology chart area
...
+        Scenario: Add to Project through drag and drop to upload JAR file in topology
+            Given user is at the Topology page
+             When user right clicks on the empty chart area
+              And user clicks on Add to Project
+              And user clicks on Upload JAR file
+              And user drag and drop the file in JAR file section
+              And user gives Application name as "sample-upload-app" and Name as "sample-yaml-upload-1"
+              And user clicks on Create
+             Then user is taken to topology
+              And user can see a toast notification of Jira file uploading with link to build logs
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature:227]:
```suggestion
              And user can see a toast notification of Jar file uploading with link to build logs
```

File: frontend/packages/dev-console/integration-tests/features/topology/topology-editing-app-node.feature [hunk 8]
```diff
@@ -37,3 +37,15 @@ Feature: Editing an application
...
+        @regression @manual
+        Scenario: Edit JAR file through drag and drop
+            Given user has uploaded JAR file
+             When user opens sidebar of the file
+              And user clicks on Edit app in Action menu
+              And user drag and drop a new JAR file in JAR file section
+              And user updates Build image version
+              And user clicks on Save
+             Then user is taken to topology
+              And user can see a toast notification of Jira file uploading with link to build logs
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/topology/topology-editing-app-node.feature:51]:
```suggestion
              And user can see a toast notification of Jar file uploading with link to build logs
```

File: frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature [hunk 9]
```diff
@@ -1,25 +1,61 @@
...
+
+        @regression, @manual
+        Scenario: Drag and drop jar file in topology list view
+            Given user has a jar file named "sample_yaml_upload.yaml"
+              And user is at the Topology list view page
+             When user drags and drop jar file on topology
+              And user gives Application name as "sample-upload-app" and Name as "sample-yaml-upload" in Upload JAR file form
+              And user clicks on Create
+             Then user is taken to topology
+              And user can see a toast notification of Jira file uploading with link to build logs
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature:37]:
```suggestion
              And user can see a toast notification of Jar file uploading with link to build logs
```

File: frontend/packages/knative-plugin/integration-tests/features/knative/create-knative-workload.feature [hunk 10]
```diff
@@ -107,3 +107,17 @@ Feature: Create a workload of 'knative Service' type resource
...
+        Scenario Outline: knative resource type in upload JAR file
+            Given user is at the Add page
+             When user clicks on the Upload JAR file card
+              And user drag and drop the file in JAR file section
+              And user selects appropriate Build image version
+              And user gives Application name as "sample-upload-app" and Name as "sample-yaml-upload"
+              And user selects resource type as "knative"
+              And user clicks on Create
+             Then user will be taken to Topology page
+              And user can see a toast notification of Jira file uploading with link to build logs
```
Comment (gajanan-more) [inline frontend/packages/knative-plugin/integration-tests/features/knative/create-knative-workload.feature:122]:
```suggestion
              And user can see a toast notification of Jar file uploading with link to build logs
```

**Review by gajanan-more — CHANGES_REQUESTED** [review]

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature (hunk 2 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:17]: I agree

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature (hunk 3 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature]: Updated

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature (hunk 4 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature]: Updated

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature (hunk 1 above)
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:1]:
```suggestion
Feature: Upload JAR file
```

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature [hunk 11]
```diff
@@ -0,0 +1,157 @@
+Feature: UploadJAR file
+              As a user, I want to upload a JAR file from Add options
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:2]:
```suggestion
              As a user, I want to upload a JAR file using Upload JAR file card from Add page
```

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+        Scenario Outline: Upload JAR file from Add options
+            Given user is at the Add page
+             When user clicks on the Upload JAR file card
+              And user clicks on Browse in JAR file section
+              And user selects file to upload
+              And user selects appropriate Build image version
+              And user gives Application name as "sample-upload-app" and workload Name as "<name>"
+              And user selects resource type as "<resource_type>"
+              And user clicks on Create
+             Then user will be taken to Topology page
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:20]:
```suggestion
             Then user will be redirected to Topology page
```

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+        @regression @manual
+        Scenario Outline: Upload JAR file from Add options
+            Given user is at the Add page
+             When user clicks on the Upload JAR file card
+              And user clicks on Browse in JAR file section
+              And user selects file to upload
+              And user selects appropriate Build image version
+              And user gives Application name as "sample-upload-app" and workload Name as "<name>"
+              And user selects resource type as "<resource_type>"
+              And user clicks on Create
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:19]:
```suggestion
              And user clicks on Create
```
```suggestion
              And user clicks create button
```

Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:19]:
```suggestion
              And user clicks  create button
```

**Review by gajanan-more — CHANGES_REQUESTED** [review]

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature (hunk 13 above)
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:19]:
```suggestion
              And user clicks  create button
```

Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:19]:
```suggestion
              And user clicks create button
```

**Review by gajanan-more — CHANGES_REQUESTED** [review]

**Review by gajanan-more — CHANGES_REQUESTED** [review]

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature (hunk 13 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:19]: We have step definition written for this statement. Make sense to use this one

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+        @regression @manual
+        Scenario Outline: Upload JAR file through drag and drop from Add options
+            Given user is at the Add page
+             When user clicks on the Upload JAR file card
+              And user drag and drop the file in JAR file section
+              And user selects appropriate Build image version
+              And user gives Application name as "sample-upload-app" and workload Name as "<name>"
+              And user selects resource type as "<resource_type>"
+              And user clicks on Create
+             Then user will be taken to Topology page
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:38]: Same as above

**Review by gajanan-more — CHANGES_REQUESTED** [review]

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature [hunk 15]
```diff
@@ -0,0 +1,157 @@
...
+
+
+        @regression
+        Scenario: Upload JAR file by unselecting options in "Build Configuration" section: A-04-TC07
+             When user is on the Upload JAR file form
+              And user clicks on Browse in JAR file section
+              And user selects file to upload
+              And user gives Application name as "sample-upload-app" and workload Name as "sample-yaml-upload"
+              And user clicks "Build Configuration" link in Advanced Options section
+              And user unselects Configure a webhook build trigger checkbox in build configuration section
```
Comment (christianvogt) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:96]: These options aren't available for upload a jar flow.

File: frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature [hunk 16]
```diff
@@ -172,3 +172,86 @@ Feature: Topology chart area
...
+              And user can see a toast notification of JAR file uploading with link to build logs
+              And user can see deployment "sample-yaml-upload-1" in application "sample-upload-app" is created in topology
+
+
+        @regression, @manual
+        Scenario: Drag and drop Incompatible file in topology chart view
+            Given user has a incompatible file
+              And user is at the Topology chart view
+             When user drags and drop the file on topology
+             Then the curser will show the action is not available
```
Comment (christianvogt) [inline frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature:236]: This is not possible. The user must first drop the file before we can validate it.

UX wants to display a toast warning message that the file is invalid.

File: frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature [hunk 17]
```diff
@@ -1,25 +1,61 @@
...
+             When user drags and drop jar file on topology
+              And user gives Application name as "sample-upload-app" and workload Name as "sample-yaml-upload" in Upload JAR file form
+              And user clicks on Create
+             Then user is redirected to topology
+              And user can see a toast notification of JAR file uploading with link to build logs
+              And user can see deployment "sample-yaml-upload" in application "sample-upload-app" is created in topology
+
+
+        @regression, @manual
+        Scenario: Drag and drop Incompatible file in topology chart view
```
Comment (christianvogt) [inline frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature:42]: Seems like there are duplicate scripts.

Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature:42]: Sorry, it's for the list view. I'll correct the scenario name

File: frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature (hunk 16 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature]: I've added about the toast warning message

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature (hunk 15 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature]: Updated

File: frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature [hunk 18]
```diff
@@ -1,25 +1,61 @@
...
+             When user clicks on View shortcuts
+             Then user sees shortcut for Drag and drop a JAR file into Topology
+
+
+        @regression, @manual
+        Scenario: Drag and drop Incompatible file in topology list view
+            Given user has a incompatible file
+              And user is at the Topology list view
+             When user drags and drop the file on topology
+             Then the curser will show the action is not available
```
Comment (christianvogt) [inline frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature:61]: Similar update needed here to show a toast.

File: frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature [hunk 19]
```diff
@@ -172,3 +172,86 @@ Feature: Topology chart area
...
+              And user clicks Clear and reupload the file
+              And user gives Application name as "sample-upload-app" and workload Name as "sample-yaml-upload-1"
+              And user clicks on Create
+             Then user is redirected to topology
+              And user can see a toast notification of JAR file uploading with link to build logs
+              And user can see deployment "sample-yaml-upload-1" in application "sample-upload-app" is created in topology
+
+
+        @regression @manual
+        Scenario: Add to Project through drag and drop to upload JAR file in topology
```
Comment (christianvogt) [inline frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature:218]: This scenario is a bit redundant with the one above it considering the difference has already been tested as part of the form tests. The interaction with topology is the same.

Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature:218]: I understand but we need to consider all the scenario as part of epic

File: frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature (hunk 18 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature:61]: Updated

File: frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature (hunk 19 above)
Comment (christianvogt) [reply frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature:218]: I suppose. I'm not sure of your overall strategy.
IMO testing topology doesn't require re-testing the different form flows. As long as one of the flows works.

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature [hunk 20]
```diff
@@ -0,0 +1,140 @@
...
+              And user can see "<resource_type>" "<name>" in application "sample-upload-app" is created in topology
+
+        Examples:
+                  | name                 | resource_type     |
+                  | sample-yaml-upload-1 | Deployment        |
+                  | sample-yaml-upload-2 | Deployment Config |
+
+        @regression @manual
+        Scenario Outline: Upload JAR file through drag and drop from Add options
+            Given user is at the Add page
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:31]: Already it is covered as part of background, please remove it

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature [hunk 21]
```diff
@@ -0,0 +1,140 @@
+Feature: Upload JAR file
+              As a user, I want to upload a JAR file from using Upload JAR file card from Add page
+
+        Background:
+            Given user is at developer perspective
+              And user is at Add page
+              And user has created or selected namespace "aut-upload-jar-file"
+
+
+        @regression @manual
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:10]: we can automate this scenario, maybe you can remove @manual and add @to-do tag

File: frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature [hunk 22]
```diff
@@ -172,3 +172,86 @@ Feature: Topology chart area
               And user hovers on Add to Project and clicks on From Channel
               And user clicks on Create
              Then user is able to see different applications created from Samples, From Git, Container Image, From Dockerfile, From Devfile, From Catalog, Database, Operator Backed, Helm Charts, Event Source, Channel
+
+
+        @regression, @manual
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature:177]: Drag and drop is possible in automation, maybe you can remove the tag @manual and add @to-do

File: frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature [hunk 23]
```diff
@@ -1,25 +1,61 @@
...
-   And user clicks on Deployment
-   And user can see only the deployment workload
-   And user clicks on Deployment-Config
-   Then user can see only the deployment-config workload
+        Background:
+            Given user is at developer perspective
+              And user has selected namespace "aut-topology-list-view"
+
+
+        @regression, @smoke
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature:9]: If it's a smoke test, please remove @regression tag

File: frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature [hunk 24]
```diff
@@ -1,25 +1,61 @@
...
+            Given user has a incompatible file
+              And user is at the Topology list view
+             When user drags and drop the file on topology
+             Then the curser will show the action is not available
+
+
+        @regression, @manual
+        Scenario: View shortcuts menu
+            Given user has uploaded a jar file
+             When user clicks on View shortcuts
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature:52]: In which page, do we see this shortcuts menu?

File: frontend/packages/knative-plugin/integration-tests/features/knative/create-knative-workload.feature [hunk 25]
```diff
@@ -107,3 +107,17 @@ Feature: Create a workload of 'knative Service' type resource
               And user clicks on the Knative Service workload "dancer-ex-git"
               And user clicks on name KSVC "dancer-ex-git"
              Then user will see value of autoscaling.knative.dev/maxScale, autoscaling.knative.dev/minScale, autoscaling.knative.dev/target, autoscaling.knative.dev/targetUtilizationPercentage, autoscaling.knative.dev/window under annotation and containerConcurrency under spec as under spec 5, 1, 3, 70, 9s and 15 respectively
+
+
+        @regression @manual
+        Scenario Outline: knative resource type in upload JAR file
```
Comment (makambalaji) [inline frontend/packages/knative-plugin/integration-tests/features/knative/create-knative-workload.feature:113]: It should be Scenario, when there is no examples

**Review by makambalaji — COMMENTED** [review]
Please add @manual tag, only if we are not able to automate the scenario 
Add the @to-do for all the scenarios wherever automation possible

File: frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature (hunk 22 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature:177]: But how do we implement it in ci

File: frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature (hunk 24 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature:52]: Topology list view page

File: frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature (hunk 22 above)
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/topology/topology-chart-area-visual.feature:177]: hmm agreed, to upload the jar file from local is not possible

File: frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature (hunk 24 above)
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature:52]: please mention that in when condition
```suggestion
             When user clicks on View shortcuts in topology list view
```

File: frontend/packages/knative-plugin/integration-tests/features/knative/create-knative-workload.feature (hunk 25 above)
Comment (sanketpathak) [reply frontend/packages/knative-plugin/integration-tests/features/knative/create-knative-workload.feature]: Updated

File: frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature (hunk 23 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/topology/topology-list-view.feature]: Updated

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature [hunk 26]
```diff
@@ -0,0 +1,140 @@
...
+        Scenario Outline: Upload JAR file from Add options
+            Given user is at the Add page
```
[the remark this answers is not in the document]
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:11]: Change the Background part

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature (hunk 20 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:30]: Change the Background part

File: frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature [hunk 27]
```diff
@@ -0,0 +1,139 @@
+Feature: Upload JAR file
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:1]: Please add @add-flow before the feature description

Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature:2]: Updated

**Pull request merged**